src/olympia/reviewers/models.py

```diff
@@ -1142,6 +1143,14 @@ def calculate_static_analysis_weight_factors(self):
             }
         return factors
 
+    def calculate_score(self):
+        """Compute maliciousness score for this version."""
+        # Some precision is lost but we don't particularly care that much, it's
```

Comment:
Ha! I guess that answers my question above :D

Reply:
Yeah, I don't think we care. It's a denormalized field, only really there because otherwise the query is nasty. And even if we were to end up using it more, I don't think the precision would matter for what we're doing with it.